pkg/virtctl/version/version.go

```diff
@@ -11,14 +14,22 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/templates"
 )
 
-var clientOnly bool
+var (
+	clientOnly                    bool
+	clientVersion                 *version.Info
+	serverVersion                 *version.Info
+	versionsNotAlignedWarnMessage = "You are using a client virtctl version that is different from the server one"
```

Comment:
Suggestion (feel free to ignore), maybe: "You are using a client virtctl version that is different from the KubeVirt version running in the cluster"

Reply:
I like it :+1: